cmd/trustbloc-agent-js-worker/main.go

```diff
@@ -17,13 +17,13 @@ import (
 
 	"github.com/google/uuid"
 	"github.com/mitchellh/mapstructure"
-	"github.com/sirupsen/logrus"
+	"github.com/trustbloc/edge-core/pkg/log"
 
```

Comment:
extra line

Reply:
Looks correct to me - can you take another look?

Reply:
not yet fixed, group trustbloc imports separately, I wonder why linter is not catching it

Reply:
"bytes"
	"encoding/json"
	"fmt"
	"syscall/js"
	"time"

	"github.com/google/uuid"
	"github.com/mitchellh/mapstructure"

	"github.com/trustbloc/edge-core/pkg/log"
	"github.com/trustbloc/edge-agent/pkg/controller"
	cmdctrl "github.com/trustbloc/edge-agent/pkg/controller/command"

Reply:
"github.com/google/uuid"
	"github.com/mitchellh/mapstructure"

	"github.com/trustbloc/edge-core/pkg/log"
	"github.com/trustbloc/edge-agent/pkg/controller"
	cmdctrl "github.com/trustbloc/edge-agent/pkg/controller/command"

Reply:
We don't group trustbloc imports separately in other projects. edge-core is an external library from edge-agent's perspective.

Here's how the imports look to me:

`	"bytes"
	"encoding/json"
	"fmt"
	"syscall/js"
	"time"

	"github.com/google/uuid"
	"github.com/mitchellh/mapstructure"
	"github.com/trustbloc/edge-core/pkg/log"

	"github.com/trustbloc/edge-agent/pkg/controller"
	cmdctrl "github.com/trustbloc/edge-agent/pkg/controller/command"`